plugins/zns/zns.c

```diff
@@ -503,6 +550,7 @@ static int set_zone_desc(int argc, char **argv, struct command *cmd, struct plug
 	OPT_ARGS(opts) = {
 		OPT_UINT("namespace-id", 'n', &cfg.namespace_id,  namespace_id),
 		OPT_SUFFIX("start-lba",  's', &cfg.zslba,         zslba),
+		OPT_FLAG("zrwa",         'r', &cfg.zrwa,          zrwa),
```

Comment:
This field is called ZRWA allocation in the specification and is specified within the "Zone Send Action Specific Option (ZSASO):" field.

Rename to "zrwaa" .